Mlem/App/Views/Root/Tabs/Settings/FiltersSettingsView.swift

```diff
@@ -12,27 +12,38 @@ struct FiltersSettingsView: View {
     @Dependency(\.persistenceRepository) var persistenceRepository
     
     @Setting(\.filters_keywordFilterEnabled) var keywordFilterEnabled
+    @Setting(\.filters_literalFilterEnabled) var literalFilterEnabled
     
     @Environment(FiltersTracker.self) var filtersTracker
     @Environment(NavigationLayer.self) var navigation
     
     @State var newKeyword: String = ""
+    @State var newLiteral: String = ""
     
     init() {
         @Dependency(\.persistenceRepository) var persistenceRepository
     }
     
     var body: some View {
         Form {
-            Section {
-                Toggle("Enable Keyword Filters", icon: .settings.keywordFilter, isOn: $keywordFilterEnabled)
-            }
+            SettingsHeaderView(
+                title: "Filters",
+                // swiftlint:disable:next line_length
+                description: "Hide posts containing certain words, phrases, or character sequences from your feed. If you are a moderator or administrator of a filtered post, it will appear in your feed but require you to tap to view its content.",
+                icon: .settings.keywordFilter
+            )
             
-            Section {
+            Section("Keywords") {
                 keywordSection
             } footer: {
                 // swiftlint:disable:next line_length
-                Text("Posts with these keywords in their titles will be hidden. If you are a moderator or administrator of a matching post, it will appear in your feed but require you to tap to view its content.")
+                Text("Hide posts with titles containing these whole words or phrases. Ignores case and punctuation (e.g., the keyword \"john\" will also filter \"John's\").")
+            }
+            
+            Section("Literals") {
+                literalSection
+            } footer: {
+                Text("Hide posts with titles containing containing these precise character sequences")
             }
         }
         .withConditionalLabelStyle()
```

Comment:
Currently there's no way to dismiss the keyboard

```suggestion
        .scrollDismissesKeyboard(.interactively)
        .withConditionalLabelStyle()
```